libraries/abstractions/ble_hal/test/ble_test_scipts/startTests_integration.py

```diff
@@ -92,6 +92,12 @@ def main():
         isTestSuccessFull,
         runTest.Check_Bond_State)
 
+    isTestSuccessFull = runTest.Change_MTU_Size(
```

Comment:
Can you please add inline commenting to all of these files?

Reply:
added